### DISCUSSION TITLE
Upgrade guide 1 -> 2

### DISCUSSION THREAD

**Discussion by allantatter** [vitobotta/hetzner-k3s#448] [Q&A] [answered]
Is there an Upgrade Guide or PR thread to go through to update `hetzner-k3s-cluster.yaml` to make my current cluster (created during v1 era) to work with hetzner-k3s v2?

**Comment by vitobotta** [comment, marked as answer]
Please use discussions for generic questions rather than technical issues with the tool :)

See release notes at https://github.com/vitobotta/hetzner-k3s/releases/tag/v2.0.0

**Reply by allantatter** [reply]
Thanks a lot. I appreciate the work you have done. Will know better next time to post in the right location.
